WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+
+\columnbreak
+
+#### Arcane Charges
+*2nd-level Arcane feature* <br>
+You gain the ability to conserve and enhance your mage abilities beyond the norm. Whenever you cast a mage spell of 1st level or higher, gain an arcane charge of the equivalent level up to 5th level. Spells slots higher than 5th give a 5th level arcane charge. These charges last until replaced by a higher level charge, used, or 1 minute has passed.
+
+You may expend an arcane charge as a bonus action to grant yourself one of the following effects. This effect lasts until the end of your next turn.
+- A bonus equal to the arcane charge's level to Attack and Damage rolls for your next Mage Spell.
+- A bonus equal to the arcane charge's level to Saving Throws against Magic or Magical Effects. You may use a reaction for this effect instead of a bonus action.
+
+#### Chrono Shift
+*2nd-level Arcane feature* <br>
+After rolling initiative, but before the first creature takes its turn, you can switch your result with the result of another creature that you can see. Once you've done so you can't use this feature again until you finish a long rest.
+
+#### Arcane Brilliance
+*6th-level Arcane feature* <br>
+You gain proficiency in the Arcana skill if you don't already have it. Your proficiency bonus is doubled for any ability check you make with it.
+
+Additionally, you are always under the effects of a *comprehend languages* spell.
+
+#### Presence of Mind
+*10th-level Arcane feature* <br>
+Whenever you create a spell slot from sorcery points you also gain an arcane charge of equivalent level and you can use it as part of the same bonus action.
+
+#### Savant
+*10th-level Arcane feature* <br>
+You can change your list of prepared mage spells when you finish a short or a long rest.
+
+#### Time Warp
+*14th-level Arcane feature* <br>
+You can exert your control over magic to warp time itself. When a creature you can see within 60 feet of you begins their turn, you can use your reaction to allow that creature to take another action during that turn. When you do so, you gain one level of exhaustion.
+
+### Study of Fire
+While any worthy mage is an unparalleled expert in the use of magic and heavily dedicated to their craft, those who master the forces of fire tend to be a little more audacious than their peers. These mages take pride, even pleasure in igniting their enemies in wild bursts of flame.
+
+#### Bonus Cantrip
+*2nd-level Fire feature* <br>
+When you choose this magical study at 2nd level, you learn the *control flames* cantrip. If you already know this cantrip, you learn a different wizard cantrip of your choice. The cantrip doesn't count against your number of cantrips known.
+
+<div class='footnote'>PART 1 | CLASSES</div>
+<img src='https://www.gmbinder.com/images/u17sxuH.jpg' style='position:absolute; top:580px; right:260px; width:700px' />
```

Comment:
I have a comment for this image relative to the one below, and that is that it monotonous to use the same page tearing style on two pages after one another. Could either one be changed, maybe flipped to the opposite corner, or similar?